src/components/ExternalLinks/EtherScanLink.js

```diff
@@ -18,7 +18,7 @@ const EtherScanLink = ({ address, tx, children }) => (
 
       let link
       if (address) {
-        link = `https://${prefix}etherscan.io/address/${address}`
+        link = `https://${prefix}.etherscan.io/address/${address}`
```

Comment:
Are you sure this is correct? The link to etherscan from https://pr121-app-wearekickback.surge.sh/event/0x083b48a2a36cccaa9e34e5e7dbf90aaec450e58f shows `rinkeby..etherscan.io/..`